controllers/calendar/CalendarController.php

```diff
@@ -78,4 +106,17 @@ public function actionFindByContainer($containerId)
         return $this->returnPagination($query, $pagination, $results);
     }
 
+    private function checkUserIsRegistered(){
+        $authHeader = Yii::$app->request->getHeaders()->get('Authorization');
+        if (!empty($authHeader) && preg_match('/^Bearer\s+(.*?)$/', $authHeader, $matches)) {
+            $token = $matches[1];
+
+            $validData = JWT::decode($token, "NFqRh33ofXCLYQ9SvFiX3lnBa7qLl2NcMBj_gYMaTCwdcxSIqY3rYxJ2UWXiE1R0Ow0oYg4fJk9HaVGykWzFry", ['HS512']);
```

Comment:
what does this token mean here? where does it come from? is it safe to hardcode it?

Reply:
this PR is an urgent topic for us, @gaurav-handysolver can you please have a look here to say something about the hardcoded token, thanks

Reply:
yes please replace the hardcoded string, thanks.

Reply:
Hi @donni106 
Done!